_config.yml

```diff
@@ -1,10 +1,13 @@
 ---
-title: perfect-fit
+title: PERFECT FIT
```

Comment:
Tiny thought: it might be better to uppercase with CSS (separating content from styling).